### DISCUSSION TITLE
Supervised classifiers

### DISCUSSION THREAD

**Discussion by johnmarkpittman** [moj-analytical-services/splink#927] [General]
There are options for using labeled data for training during the pairwise classification step - have y'all considered allowing users to use other classification algorithms? For example, `dedupe` allows you to use any sklearn-style classifier that has a `predict_proba` method.

**Comment by RobinL** [comment]
It's a valid approach but not one we're intending to include in Splink, for a few reasons:
- it's not clear to me what the advantage would be over just using raw scikit
- It would potentiallybe challenging to scale to big data
- it's rare to have a large dataset of high quality labels
- my gut is that the improvement in accuracy would be modest
